src/classes/window/fullscreen.vala

```diff
@@ -105,12 +105,14 @@ namespace pdfpc.Window {
             var icon_theme = Gtk.IconTheme.get_default();
             // Fallback, if absent in the default icon theme
             icon_theme.append_search_path(Resources.resolve("icons"));
+#if !WIN32
```

Comment:
What would happen if windows is loading this piece of code? Does it hurt to also load the icon on Windows? Or is it interfering with the icon shipped with the windows installation?

Reply:
Just noticed that this isn't needed. I ran the program without the new resources installed, where it complained about not finding the icon. Running it properly doesn't print anything.